lx_anonymizer/sensitive_meta_interface.py

```
@@ -0,0 +1,34 @@
+    examiner_first_name: Optional[str] = None
+    examiner_last_name: Optional[str] = None
+    center: Optional[str] = None
+
+    def __getitem__(self, key: str) -> Any:
+        return getattr(self, key)
+
+    def __setitem__(self, key: str, value: Any) -> None:
+        if hasattr(self, key):
+            setattr(self, key, value)
+        else:
+            raise KeyError(f"Invalid key '{key}' for SensitiveMeta")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)